lib/deploy.rb

```diff
@@ -22,24 +24,42 @@ def run
       check_for_unstaged_changes!
       check_for_changelog!
       set_aws_region!
+      fetch_eb
       verify_configuration!
 
-      @name, @app_bucket = select_app
-      @platform          = detect_platform
+      @name       = deployment_target
+      @platform   = detect_platform
 
       request_confirmation!
       synchronize_repo!
       deploy!
```

Comment:
Though this is not a change behaviour, I find it worrying that just initialising an object would trigger a deployment, instead of explicitly calling the ``deploy!`` method